app/utils/decorator_utils.py

```diff
@@ -0,0 +1,43 @@
+"""
+This module is used to define decorators for the app
+"""
+from app.database.models.user import UserModel
+
+
+def email_verification_required(user_function):
+    """
+    This function is used to validate the input function i.e. user_function
```

Comment:
According to PEP8 docstrings and comments line lengths should be 72 characters: https://www.python.org/dev/peps/pep-0008/#id17.